app/src/main/java/com/abominableshrine/taptounlock/MainActivity.java

```diff
@@ -25,20 +25,20 @@ protected void onCreate(Bundle savedInstanceState) {
 
         // Initialize the button that toggles the service on or off
         serviceStatus = (TextView) findViewById(R.id.service_status);
-        serviceStatus.setText(SensorListenerService.isRunning() ? R.string.service_running : R.string.service_not_running);
-        sensorReadingServiceIntent = new Intent(this, SensorListenerService.class);
+        serviceStatus.setText(UnlockService.isRunning() ? R.string.service_running : R.string.service_not_running);
```

Comment:
This might be helpful: Checkout [isServiceRunning](https://github.com/psycholold/TapToUnlock/commit/c5b50be66044824b34abc89aaea57903f62d43b4#diff-6d57de4d042d8e28a8bb12fbcd976a13R25) method for a more generic way to check if a service is running :-)